### ISSUE TITLE
Release missing

### ISSUE THREAD

**Issue by walter-weinmann** [mikra01/nimodpi#5]
We are planning to include Nim in our [OraBench project](https://github.com/KonnexionsGmbH/ora_bench) and therefore need versioning of nimodpi. 

Thank you for your support.!

**Comment by mikra01** [comment]
yep - at the moment there is no initial release present - I'll plan to do a release when lob-support is functional..
